src/main/java/net/sf/jabref/external/push/PushToApplicationButton.java

```diff
@@ -56,9 +54,10 @@
     private final JPopupMenu optPopup = new JPopupMenu();
     private final JMenuItem settings = new JMenuItem(Localization.lang("Settings"));
 
-    public PushToApplicationButton(JabRefFrame frame, List<PushToApplication> pushActions) {
-        this.frame = frame;
-        this.pushActions = pushActions;
+
+    public PushToApplicationButton(JabRefFrame frameJR, List<PushToApplication> pushActionsList) {
+        frame = frameJR;
+        pushActions = pushActionsList;
```

Comment:
I see the difference between these cases (assigning in the constructor/init). There was also a case discussed where, translated to this case, `PushToApplicationButton.frame` was discouraged. I just interpolated to this, but no problem to remove it. Not sure what is best practice (there are quite a few places where things like this occur in the constructor, so probably better not touch it and restore this one).